geniza/corpus/metadata_export.py

```diff
@@ -193,6 +205,12 @@ class FragmentExporter(Exporter):
         "last_modified",
     ]
 
+    # queryset filter for content types included in this import
+    content_type_filter = {
+        "content_type__app_label__in": ["corpus"],
+        "content_type__model__in": ["document", "fragment", "collection"],
+    }
+
```

Comment:
This is a nice solution to make sure we're exporting the right docs and objects.

Reply:
yeah, but it may be hard to keep it updated if we modify the exports and include data from other models